src/layout/Border.js

```diff
@@ -72,11 +79,11 @@
         switch(this.layoutType()){
             default:
                 switch(sectionType){
-                    case "topSection":return ["s"];
-                    case "rightSection":return ["w"];
-                    case "bottomSection":return ["n"];
-                    case "leftSection":return ["e"];
-                    case "centerSection":return [];
+                    case "top":return ["s"];
+                    case "right":return ["w"];
+                    case "bottom":return ["n"];
+                    case "left":return ["e"];
+                    case "center":return [];
```

Comment:
This is technically a breaking change - but suspect its ok - but in future we need to be careful about things like this.